.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: pyronear
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
I only suggest renaming it DOCKERHUB_PW or DOCKERHUB_PASSWORD (there are several docker registries, one on github by the way and we may, in the future, push the image to multiple repositories)

Reply:
Well spotted, thanks !